spec/features/merchant/dashboard_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe "merchant dashboard" do
+  it "should show the merchant user's info" do
+    merchant_employee = create(:merchant_employee)
+
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(merchant_employee)
+
+    visit "/merchant"
+
+    expect(page).to have_content merchant_employee.name
+    expect(page).to have_content merchant_employee.address
+    expect(page).to have_content merchant_employee.city
+    expect(page).to have_content merchant_employee.state
+    expect(page).to have_content merchant_employee.zip
```

Comment:
Looks great! Im not sure if I read the story wrong but i think 34 wants the employers?

Reply:
employer information*

Reply:
Oh yep, you're right! `merchant_employee` is a User class. I'd need to pull the merchant info. Good catch

Reply:
Made the changes and pushed them up. Let me know how it looks now

Reply:
Great Job!